reporter/src/main/kotlin/reporters/TableReporter.kt

```diff
@@ -222,7 +222,7 @@ abstract class TableReporter : Reporter() {
             val error: OrtIssue,
             val resolutions: List<ErrorResolution>
     ) {
-        override fun toString() =
+        fun getDescription(): String  =
```

Comment:
The function always returns the same result, so if you do not override `toString()` anymore you could also make it a `val`.